elasticsearch_collectd.py

```diff
@@ -1016,9 +1082,15 @@ class CollectdMock(object):
     def __init__(self):
         self.value_mock = CollectdValuesMock
 
+    def debug(self, msg):
+        print'DEBUG: {}'.format(msg)
```

Comment:
@amreshakim Good catch, I have corrected the issue and squashed the commit.  Thanks!